libs/barista-components/stacked-series-chart/README.md

```diff
@@ -119,11 +120,11 @@ This `DtStackedSeriesChartSeries` holds the information for one series.
 This `DtStackedSeriesChartNode` holds the information for every node in a given
 series.
 
-| Name    | Type                | Optional | Description                                                                                             |
-| ------- | ------------------- | -------- | ------------------------------------------------------------------------------------------------------- |
-| `label` | `string`            | No       | Name of the node to be shown.                                                                           |
-| `value` | `number`            | No       | Numeric value used to calculate the slices.                                                             |
-| `color` | `DtColors | string` | Yes      | Color to be used. Fallback to [sorted chart colors](/resources/colors/chartcolors#sorted-chart-colors). |
+| Name    | Type      | Optional | Description                                 |
+| ------- | --------- | -------- | ------------------------------------------- |
+| `label` | `string`  | No       | Name of the node to be shown.               |
+| `value` | `number`  | No       | Numeric value used to calculate the slices. |
+| `color` | `DtColors | string`  | Yes                                         | Color to be used. Fallback to [sorted chart colors](/resources/colors/chartcolors#sorted-chart-colors). |
```

Comment:
Something weird happened with the formatting. Have you checked it's correct in the docs?

Reply:
Yes, it looks like there is some git hook that is autoformatting the code, I've tried with two different IDEs(VSCode and Intellij) and even without applying the autoformatter, but once you commit it the format changes to the new one

Reply:
The formatter is called prettier and is running on as pre-commit hook. Prettier formats markdown tables based on the length of its content.

Reply:
ThenI guess it's fine, I saw this nx format:write that was run before each commit